modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -22,23 +20,11 @@ pub trait WalletClientExt {
 
 impl WalletClientExt for Client {
     fn get_network(&self) -> Network {
-        let (_, wallet_client) = wallet_client(self);
-        wallet_client.get_network()
+        let (wallet, _instance) = self.get_first_module::<WalletClientModule>(&KIND);
```

Comment:
TODO for later: add `KIND` to client module trait such that it deosn't need to be supplied here